packages/nx-go/src/utils/go-bridge.ts

```diff
@@ -99,6 +100,14 @@ export const createGoWork = (tree: Tree): void => {
   }
 };
 
+export const isProjectUsingNxGo = (targets: {
```

Comment:
This method is not related to Go CLI or files, so it shouldn’t be in this file, but we can leave it as is for now